.github/workflows/release-prod.yml

```diff
@@ -15,6 +15,11 @@ jobs:
     name: Release
     if: ${{ github.repository == 'clerkinc/javascript' }}
     runs-on: ${{ vars.RUNNER_LARGE }}
+    env:
+      NPM_CONFIG_PROVENANCE: true
```

Comment:
💡 We might be able to add `provenance=true` to the root `.npmrc` file in the repo, instead of needing to add the environment variable to each release workflow.

Reply:
I tried that in [`b6c514a` (#1891)](https://github.com/clerkinc/javascript/pull/1891/commits/b6c514a2fffc4e08841a9d0d9a53f1973d9c403b) but Verdaccio didn't like that: https://github.com/clerkinc/javascript/actions/runs/6533197979/job/17737954907?pr=1891

Reply:
@LekoArts It looks like that error might have been because the `id-token` permission wasn't set yet 👀

Reply:
I tried that in [`3545ced` (#1891)](https://github.com/clerkinc/javascript/pull/1891/commits/3545ced3ef0a968329fc9f57426cae1bcb95eb8f), that [didn't make a difference](https://github.com/clerkinc/javascript/actions/runs/6534106471/job/17740703222?pr=1891)